### ISSUE TITLE
VS Code: we have marked your extension as deprecated

### ISSUE THREAD

**Issue by isidorn** [reasonml-editor/vscode-reasonml#296]
Hi VS Code PM here 👋 

We have marked your extension as deprecated. If that is not the case please do let us know and we can easily change that

Thank you!

**Comment by EliasGit2017** [comment]
This extension is used by many Ocaml devs, please get it back ASAP

**Comment by TheSpyder** [comment]
I don’t believe there are that many. And anyone new to the language should be starting with the ocaml platform extension, not this one.
